tests/pdctl/helper.go

```diff
@@ -53,6 +54,7 @@ func CheckStoresInfo(c *check.C, stores []*api.StoreInfo, want []*metapb.Store)
 	}
 	for _, s := range stores {
 		obtained := proto.Clone(s.Store.Store).(*metapb.Store)
+		obtained.State = metapb.StoreState(metapb.StoreState_value[s.Store.StateName])
```

Comment:
`StateName` contains `Down` and `Disconnected` which are not contained in `StoreState_value`.

Reply:
I think it's a way to declare a new map for  `StoreState_value`